README.md

```diff
@@ -44,29 +44,29 @@ Install a newer MSI
 
 #### Linux/BSD/MacOSX/POSIX - Build from Source
 
-- **Glide**
+- **Dep**
 
-  Glide is used to manage project dependencies and provide reproducible builds.
+  Dep is used to manage project dependencies and provide reproducible builds.
   To install:
 
-  `go get -u github.com/Masterminds/glide`
+  `go get -u github.com/golang/dep/cmd/dep`
 
-Unfortunately, the use of `glide` prevents a handy tool such as `go get` from
+Unfortunately, the use of `dep` prevents a handy tool such as `go get` from
 automatically downloading, building, and installing the source in a single
 command.  Instead, the latest project and dependency sources must be first
-obtained manually with `git` and `glide`, and then `go` is used to build and
+obtained manually with `git` and `dep`, and then `go` is used to build and
 install the project.
 
 **Getting the source**:
 
 For a first time installation, the project and dependency sources can be
-obtained manually with `git` and `glide` (create directories as needed):
+obtained manually with `git` and `dep` (create directories as needed):
 
 ```
 git clone https://github.com/decred/dcrd $GOPATH/src/github.com/decred/dcrd
 cd $GOPATH/src/github.com/decred/dcrd
-glide install
-go install $(glide nv)
+dep ensure
+go install ./...
```

Comment:
We probably want to use `go install . ./cmd/...` here and below so it ignores vendor on older go versions.